src/session/sidebar/row.rs

```diff
@@ -1,46 +1,48 @@
 use gettextrs::gettext;
-use gtk::{glib, prelude::*, subclass::prelude::*};
+use gtk::{glib, prelude::*, subclass::prelude::*, CompositeTemplate};
 use std::borrow::Cow;
 use tdgrand::enums::{CallDiscardReason, ChatType, MessageContent};
 use tdgrand::types::MessageCall;
 
 use crate::session::chat::{BoxedChatNotificationSettings, Message, MessageSender};
-use crate::session::components::Avatar;
+use crate::session::sidebar::Avatar;
 use crate::session::{BoxedScopeNotificationSettings, Chat, Session, User};
 use crate::utils::{dim_and_escape, escape, human_friendly_duration};
 
 mod imp {
     use super::*;
-    use adw::subclass::prelude::BinImpl;
-    use gtk::CompositeTemplate;
     use once_cell::sync::Lazy;
     use std::cell::RefCell;
 
-    use crate::session::sidebar::Avatar as SidebarAvatar;
-
     #[derive(Debug, Default, CompositeTemplate)]
-    #[template(resource = "/com/github/melix99/telegrand/ui/sidebar-chat-row.ui")]
-    pub struct ChatRow {
-        pub chat: RefCell<Option<Chat>>,
+    #[template(resource = "/com/github/melix99/telegrand/ui/sidebar-row.ui")]
+    pub struct Row {
+        pub item: RefCell<Option<glib::Object>>,
```

Comment:
Here, a comment what sub types are expected would be helpful to better understand the code. Or would it even better to use an enum here?

Reply:
> Or would it even better to use an enum here?

I was going with that solution at first, but then I realized that we set the item from the GtkListItem (which is a GObject), so in that case I would still have to check if the type is correct, so it's basically useless. Because of that, I decided to go with the first suggestion of just adding a comment.